lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -58,7 +58,6 @@ class DeployTask
     # core/v1/Endpoints -- managed by services
     # core/v1/PersistentVolumeClaim -- would delete data
```

Comment:
I wonder if we should re-assess this choice as well. It is true that deleting a PVC can result in data loss (depending on the persistentVolumeReclaimPolicy on the underlying PV), but failing to prune them can result in continuing costs for undesired, unused storage.

Reply:
Oooo good call! I'll do some more digging.